addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -77,6 +77,9 @@ def _l10n_in_edi_extract_digits(self, string):
         result = "".join(matches)
         return result
 
+    def _l10n_in_is_global_discount(self, line):
+        return line.product_id and not line.tax_ids and line.company_id == line.product_id or False
```

Comment:
```suggestion
        return line.product_id and not line.tax_ids and **line.company_id == line.product_id** or False
```
How does that happen?